dockerfiles/rapids.Dockerfile

```diff
@@ -53,15 +53,16 @@ if [[ "$USE_FISH_SHELL" == "YES" ]]; then \
     apt install -y fish; \
 fi' \
  && bash -c '\
-if [[ "$CUDA_SHORT_VERSION" == "10.1" ]]; then \
-    apt install -y cuda-nsight-systems-10-1 nsight-systems-2019.3.7  \
-                   cuda-nsight-compute-10-1 nsight-compute-2019.4.0; \
-elif [[ "$CUDA_SHORT_VERSION" == "10.2" ]]; then \
-    apt install -y cuda-nsight-systems-10-2 nsight-systems-2019.5.2  \
-                   cuda-nsight-compute-10-2 nsight-compute-2019.5.0; \
-elif [[ "$CUDA_SHORT_VERSION" == "11.0" ]]; then \
+if [[ "$CUDA_SHORT_VERSION" == "11.0" ]]; then \
     apt install -y cuda-nsight-systems-11-0 nsight-systems-2020.2.5  \
                    cuda-nsight-compute-11-0 nsight-compute-2020.1.2; \
+elif [[ "$CUDA_SHORT_VERSION" == "11.1" ]]; then \
+    apt install -y cuda-nsight-systems-11-1 nsight-systems-2020.2.5  \
+                   cuda-nsight-compute-11-1 nsight-compute-2020.2.1; \
+elif [[ "$CUDA_SHORT_VERSION" == "11.2" ]]; then \
+    apt install -y cuda-nsight-systems-11-1 nsight-systems-2020.2.5  \
+                   cuda-nsight-compute-11-1 nsight-compute-2020.3.0; \
+
```

Comment:
We still need to change this to install the "latest" version of these which should work across any CUDA toolkit version. @jrhemstad I think you had the link to the page with the correct nsight install instructions?

Reply:
Jake mentioned the nsight versions aren't tied to specific cuda versions. I'm guessing we can merge as is without issue.

Reply:
At a minimum this won't work because `11.2` needs to be `11.2.0`, but I'll do some experimenting locally and see if I can remove these conditionals and install a single version.

Reply:
Thanks